videostore/videostore.go

```diff
@@ -452,14 +478,47 @@ func (vs *videostore) deleter(ctx context.Context) {
 			return
 		case <-ticker.C:
 			// Perform the deletion of the oldest clip
-			if err := vs.segmenter.cleanupStorage(); err != nil {
+			if err := cleanupStorage(vs.config.Storage.StoragePath, vs.config.Storage.SizeGB, vs.logger); err != nil {
 				vs.logger.Error("failed to clean up storage", err)
 				continue
 			}
 		}
 	}
 }
 
+func cleanupStorage(storagePath string, maxStorageSizeGB int, logger logging.Logger) error {
+	maxStorageSize := int64(maxStorageSizeGB) * gigabyte
```

Comment:
Could this be zero or negative and what will it do if so?

Reply:
it can't be negative or zero b/c you can only exectue this code if you created a VideoStore and you only can create a VideoStore through the constructors and the constructors take a videostore.Config and call Validate on it before instantiating a video store.

videostore.Config.Validate will valdiate that this is positive https://github.com/viam-modules/video-store/blob/main/videostore/config.go#L92